app/packages/core/src/components/Filters/NumericFieldFilter.tsx

```diff
@@ -167,6 +168,9 @@ const NumericFieldFilter = ({
   });
 
   const isFiltered = useRecoilValue(fos.fieldIsFiltered({ modal, path }));
+  const hasVisibilitySetting = useRecoilValue(
```

Comment:
Letting visibility setting be part of `fos.fieldIsFiltered` flag selector would avoid any changes to this component. Might be cleaner

Reply:
![Screenshot 2023-07-13 at 2 26 59 PM](https://github.com/voxel51/fiftyone/assets/17770824/5d4ee85f-38e4-4ba6-a4ec-1570cc2f7086)
Thought about this, but `fieldIsFiltered` atom is also used in the `numGroupFieldsFiltered` selectorFamily to calculate these status icons. We want to have separate active filter and visibility counts, soI kept separate atoms